README.md

```diff
@@ -0,0 +1,57 @@
+# Software Developers Global Summit 2020
+Annual conference for developers around the world.
+
+## Project Design:
+
+![screenshot](/images/page-screenshot.PNG)
```

Comment:
- [ ] The screenshot you have put here does not reflect your page. This one has an image background from shutter-stock yet your project page does not have background image. Kindly update the screenshot image. :+1:

Reply:
Hi may you kindly help me out with the video

Reply:
I don't how to put it